store/tikv/rawkv.go

```diff
@@ -298,3 +322,106 @@ func (c *RawKVClient) sendDeleteRangeReq(startKey []byte, endKey []byte) (*tikvr
 		return resp, actualEndKey, nil
 	}
 }
+
+func (c *RawKVClient) sendBatchPut(bo *Backoffer, keys, values [][]byte) error {
+	keyToValue := make(map[string][]byte)
+	for i, key := range keys {
+		keyToValue[string(key)] = values[i]
+	}
```

Comment:
It seems that this map can be optimized if you use the index to refer keys and values, through you may need to apply some changes to _GroupKeysByRegions_.

Reply:
Other invoking `GroupKeysByRegions` are only used to group keys. Now we can just optimize the `keyToValue` map by using the index to refer key and value.

Reply:
okay. We may keep it like this and optimize it in the future.